frontend/fingertips-frontend/.env.development

```diff
@@ -1 +1 @@
-FINGERTIPS_API_URL="http://localhost:5144"
+FINGERTIPS_API_URL="http://localhost:5144/weatherforecast"
```

Comment:
I know I suggested doing this when you were seeing issues, but thinking about it, we should really be leaving this value alone and including the `/weatherforecast` path in the page(s) that want weather forecast data. Otherwise we're going to stop ourselves from being able to use this URL as the base for any other API queries.